backend/src/main/resources/db/migration/V2__create_audit_tables.sql

```diff
@@ -127,22 +127,22 @@ ALTER TABLE ONLY selfexamination_record_aud
     ADD CONSTRAINT selfexamination_record_aud_pkey PRIMARY KEY (id, rev);
 
 ALTER TABLE ONLY examination_record_aud
-    ADD CONSTRAINT fk5lxlg92fn2q6mul14ly8n5fbr FOREIGN KEY (rev) REFERENCES revinfo(rev);
+    ADD CONSTRAINT exam_record_aud_pkey FOREIGN KEY (rev) REFERENCES revinfo(rev);
```

Comment:
Maybe fk_ prefix? It is not necessary, I am just used to it. Some conventions :). pk_ for primary key. unique_ for unique and so on.

Reply:
Sure, done in [a1ee3d7](https://github.com/cesko-digital/loono-be/pull/73/commits/a1ee3d78f5ee63ab2e3614fc90b8ff8544c110ce)